css/layout.css

```diff
@@ -0,0 +1,71 @@
+.container {
+  margin: 0 auto;
+  padding: 0 4.5rem;
+  max-width: 600px;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.column-mobile-full {
+  width: 100%;
+}
+
+.column-one-third {
+  width: calc(100% / 3);
+}
+
+.column-two-third {
+  width: calc(200% / 3);
+}
+
+@media screen and (min-width: 750px) {
+  .container {
+    max-width: 800px;
+  }
+
+  .column-desktop-half {
+    width: 50%;
+  }
+
+  .padding-right {
+    padding-right: 2.5rem;
+  }
+}
```

Comment:
Media queries should always be the last thing in any stylesheet. Media queries are reliant on Source Order to do their job, so if other styling is placed below it, it can ruin the functionality of that media query.

Reply:
Noted Noted! Thank you for the feedback